src/main/java/cz/xtf/openshift/OpenShiftUtil.java

```diff
@@ -883,6 +883,7 @@ public void cleanAndAssert() {
 	public void clean() {
 		// keep the order for deletion to prevent K8s creating resources again
 		client.extensions().deployments().delete();
+		client.apps().statefulSets().delete();
```

Comment:
You should also extend `OpenShiftWaiters::isProjectClean` so it checks stateful sets to be deleted

Reply:
Done!